DOCSUP-4280: Update the SELECT query (#17231)
ClickHouse/ClickHouse: docs/en/sql-reference/statements/select/index.md

For more information, see the section “Settings”. It is possible to use external sorting (saving temporary tables to a disk) and external aggregation.

## SELECT modifiers {#select-modifiers}

> You can modify the results returned from a `SELECT` query, as follows.

Reviewer: ```suggestion
You can use the following modifiers in `SELECT` queries.
```

```

```
┌─sum(i)─┬─sum(j)─┬─sum(k)─┐
│    220 │     18 │    347 │
└────────┴────────┴────────┘
```

### EXCEPT {#except-modifier}

Specifies the names of one or more columns to exclude from the result. All matching column names are omitted from the output.

**Syntax:**

``` sql
SELECT <expr> EXCEPT ( col_name1 [, col_name2, col_name3, ...] ) FROM [db.]table_name
```

**Example:**

``` sql
SELECT * EXCEPT (i) from columns_transformers;
```

```
┌──j─┬───k─┐
│ 10 │ 324 │
│  8 │  23 │
└────┴─────┘
```

### REPLACE {#replace-modifier}

Specifies one or more [expression aliases](../../../sql-reference/syntax.md#syntax-expression_aliases). Each alias must match a column name from the `SELECT *` statement. In the output column list, the column that matches the alias is replaced by the expression in that `REPLACE`.

This modifier does not change the names or order of columns. However, it can change the value and the value type.

**Syntax:**

``` sql
SELECT <expr> REPLACE( <expr> AS col_name) from [db.]table_name
```

**Example:**

``` sql
SELECT * REPLACE(i + 1 AS i) from columns_transformers;
```

```
┌───i─┬──j─┬───k─┐
│ 101 │ 10 │ 324 │
│ 121 │  8 │  23 │
└─────┴────┴─────┘
```

> ### Use Cases {#modifiers-use-cases}

Reviewer: ```suggestion
### Combinations of Modifiers {#modifiers-combinations}
```
